mlrun/api/main.py

```diff
@@ -1,6 +1,8 @@
 import uvicorn
-from fastapi import FastAPI
+from fastapi import FastAPI, Request, HTTPException
```

Comment:
Strong suggest to just import `fastapi`
So the error types will be clear (+namespacing)
for example line 47 will be
```
request, fastapi.HTTPException(status_code=status_code, detail=error_message)
```